flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_WindPage.js

```diff
@@ -202,7 +241,7 @@ class CDUWindPage {
         }
 
         const altitude = parseInt(elements[2]);
-        if (!isFinite(altitude) || altitude < 0 || altitude > 390) {
+        if (!isFinite(altitude) || altitude < 0 || altitude >= 450) {
```

Comment:
I think an entry of FL 450 is legit.

```suggestion
        if (!isFinite(altitude) || altitude < 0 || altitude > 450) {
```